spk/urbackup/src/wizard/install_uifile

```diff
@@ -0,0 +1,15 @@
+[{
+    "step_title": "Urbackup",
+    "items": [{
+	"type": "textfield",
+        "desc": "Urbackup backup location. At this location client backup files are saved",
+        "subitems": [{
+            "key": "wizard_backup_dir",
+            "desc": "Backup path",
+            "defaultValue": "/mnt/backups/urbackup",
```

Comment:
I would propose /volume1/urbackup as default location on DSM. FAQ would invite to create a Share Folder named "urbackup". Advanced users may select an other volume according to their own setup.

Reply:
ok